### ISSUE TITLE
Task: Update X (Twitter) link previews to match latest design

### ISSUE THREAD

**Issue by matt-west** [Automattic/wp-calypso#84430]
### Details

X has recently updated the design of link previews. How we present X previews under the **Search & Social** section of the previewer is now outdated.

### Current implementation:

<img width="1141" alt="Screenshot 2023-11-22 at 15 34 33" src="https://github.com/Automattic/wp-calypso/assets/417538/353095eb-1dab-4da6-81e7-1bf527d86728">

### New X link preview design:
![image](https://github.com/Automattic/wp-calypso/assets/417538/d35278bb-b250-4e5d-a313-c8e567796dde)
Note the lack of page title and description. There's also a new bookmark icon in the bottom right of the card.

### Checklist

- [ ] Remove site title and description
- [ ] Update style and positioning of site URL
- [ ] Add bookmark icon to lower right of card

**Comment by matt-west** [comment]
> In an upcoming release, 𝕏 will overlay title in the upper potion of the image of a URL card
https://twitter.com/elonmusk/status/1727500747711852560

Looks like this is liable to change again soon.

**Comment by manzoorwanijk** [comment]
Yes, Twitter seems to be playing with different designs for the preview. So, I think we should give it some more time to avoid a wasted effort.

**Comment by manzoorwanijk** [comment]
There are two types of previews Twitter uses

<img width="614" alt="Image" src="https://github.com/user-attachments/assets/6aaa4e3f-ea2c-4f0a-83de-fd35142b3ad3" />

The first one has

```html
<meta name="twitter:card" content="summary_large_image">
```

The second one has 

```html
<meta name="twitter:card" content="summary">
```

How should we deal with that?

**Comment by manzoorwanijk** [comment]
Just to note it here. Jetpack uses `summary_large_image` for the OG tags for twitter. So, we can implement that preview.
